index.js

```diff
@@ -0,0 +1,115 @@
+
+
+$(document).ready(()=>{
+
+
+  const successCallback = function(response) {
+    console.log("POST request was successful");
+    console.log(response);
+
+    let generatedHTML = '<p>Everything went great: ';
+    generatedHTML += `${response.name} is reserved for trip ID: ${ response.trip_id }.</p>`;
+
+      $('#reservation-results').hide().show('slow').html(generatedHTML);
+      setTimeout(function(){ $('#reservation-results').hide(); }, 10000);
+
+  };
+
+
+  let loadTrips = function loadTrips() {
+
+  $.get('https://trektravel.herokuapp.com/trips',
+    (response) => {
+      console.log('success!');
+      console.log(response);
```

Comment:
Since this success callback is so long, it might make sense to break it out into a named function. You'll have to call it something other than `successCallback`, since that name is already taken.